packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -454,6 +454,30 @@ export class DataStores implements IDisposable {
         return builder.getGCData();
     }
 
+    /**
+     * After GC has run, called to notify this Container's data stores of routes that are used in it.
+     * @param usedRoutes - The routes that are used in all data stores in this Container.
+     */
+    public updateUsedRoutes(usedRoutes: string[]) {
+        // Build a map of data store ids to routes used in it.
+        const usedRoutesMap: Map<string, string[]> = new Map();
+        for (const route of usedRoutes) {
+            const dataStoreId = route.split("/")[1];
```

Comment:
May want to assert here (fail-fast) if route is not of form "/...", because here we are assuming this structure.